src/replaykit.cs

```diff
@@ -203,9 +204,12 @@ interface RPBroadcastConfiguration : NSCoding, NSSecureCoding {
 		[Export ("clipDuration")]
 		double ClipDuration { get; set; }
 
-		// TODO review, once API documentation is available, if we can provide an easier to use strongdictionary
 		[NullAllowed, Export ("videoCompressionProperties", ArgumentSemantic.Strong)]
-		NSDictionary<NSNumber, INSSecureCoding> VideoCompressionProperties { get; set; }
+#if XAMCORE_2_0
```

Comment:
from beta 1
> @property (nonatomic, strong, nullable) NSDictionary <NSNumber *, NSObject <NSCoding, NSSecureCoding> *> *videoCompressionProperties;

say our tools don't catch such changes :( and it seems we missed the diff from a later beta